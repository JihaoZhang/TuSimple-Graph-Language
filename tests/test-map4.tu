int main()
{	
	map @{string, int} m;
  	new m;

  	m.put("hello", 1);
    m.put("hello", 2);
    m.put("hi", 4);
    m.put("hello", 3);


  	print(m.get("hello"));

  	print(m.size());

  	if (m.haskey("hello")) prints("true");
  	else prints("false");

  	m.remove("hello");

  	if (m.haskey("hello")) prints("true");
  	else prints("false");
  	
  	print(m.size());


 	return 0;
} 