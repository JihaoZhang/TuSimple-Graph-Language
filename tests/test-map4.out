3
2
true
false
1